modules/renter/repair.go

```diff
@@ -126,7 +126,11 @@ func (r *Renter) addFileToRepairState(rs *repairState, file *file) {
 		// Check whether this contract is offline. Even if the contract is
 		// offline, we want to record that the chunk has attempted to use this
 		// contract.
-		offline := r.hostContractor.IsOffline(contract.ID)
+		var offline bool
+		renterContract, exists := r.hostContractor.Contract(contract.ID)
+		if exists && renterContract.InGoodStanding {
+			offline = true
+		}
```

Comment:
`offline := exists && renterContract.InGoodStanding`

wait, why is it marked offline if the contract is in good standing?